blog/content/post/2020-04-26-measuring_memory_usage_rte.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Measuring memory and CPU usage on Yocto & Armbian
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/rte-new-revision.jpg
+author: piotr.konkol
+layout: post
+published: true
+date: 2020-04-26
+archives: "2020"
+
+tags:
+  - measurment
+  - rte
+  - yocto
+  - armbian
+  - ram
+categories:
+  - IoT
+  - Miscellaneous
+
+---
+
+## Intro
+This article will show you how to perform and interpret measurements of some of
+embedded systems computational resources. Case covered here will be an attemt
+to resolve a question whether our configuration will be enough for the given
+tasks, or an upgrade of the hardware specification will be necessary. As IoT and
+embedded systems often get very big in terms of device number, it is necessary
+to perfrom such analysis and match the specification exactly with the planned
+workload and the profit from such action will show as the system grows larger.
+
+Hardware configuration we will use is based on [RTE](https://3mdeb.com/products/open-source-hardware/rte/)
+with Orange Pi Zero 256MB. This is the default configuration with which it
+is shipped. Question may be made, whether 256MB of memory is sufficient, or
+should alternative, 512MB Orange Pi Zero be worth upgrading to. We will compare
+usage of memory on two operating systems - Armbian and Yocto. Memory will be
```

Comment:
@pkonkol link to Yocto meta layer and binary used would be useful, as well as the precise version and link to binary of Armbian

Reply:
@artur-rs Do you know if I could find such files somewhere?

Reply:
@pkonkol links sent in direct msg